packages/components/copybutton/src/CopyButton.styles.ts

```diff
@@ -1,12 +1,27 @@
 import { css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
+import type { CSSObject } from '@emotion/serialize';
 
-export const getStyles = () => ({
+const sizeToStyles = ({ size }): CSSObject => {
```

Comment:
Just a nit pick suggestion
IMO naming the function with `getSizeStyles` would make it easier to understand what it is doing

what did you mean with sizeTOstyles? What is this "to" in the name exactly?